DOCSP-41669 release notes week ending 7/26 (#3341)
mongodb/docs-realm: docs-release-notes.md

# July 26, 2024

## Flutter SDK

- Sync Device Data: Add information about converting a non-synced realm to a synced realm.

## .NET SDK

> - Sync Data: Update pages to reflect that converting to a Synced Realm is only supported in Partition Bases Sync.

Author: ```suggestion
- Sync Data: Update pages to reflect that converting to a Synced Realm is only supported with Partition Based Sync.
```

> - Sync Data: Update pages to reflect that converting to a Synced Realm is only supported with Partition Based Sync.

Author: ```suggestion
- Sync Data: Update pages to reflect that converting from a non-synced realm to a synced realm is only supported with Partition Based Sync.
```

> - Sync Data: Update pages to reflect that converting from a non-synced realm to a synced realm is only supported with Partition Based Sync.

Author: ```suggestion
- Sync Data: Update pages to reflect that converting a non-synced realm to a synced realm is only supported with Partition Based Sync.
```

> - Sync Data/Suspend or Resume a Sync Session: Add additional context to the existing code snippet.

Reviewer: ```suggestion
- Sync Data/Suspend or Resume a Sync Session: Add additional context to the existing code snippet.
```
Can you add what section this code snippet was in or what it was for?

> - Sync Data: Update pages to reflect that converting a non-synced realm to a synced realm is only supported with Partition Based Sync.
> - Sync Data/Suspend or Resume a Sync Session: Add additional context to the existing code snippet.

Reviewer: ```suggestion
- Sync Data: 
  - Suspend or Resume a Sync Session: Add additional context to the existing code snippet.
  - Throughout the section: Update pages to reflect that converting a non-synced realm to a synced realm is only supported with Partition Based Sync.
```
Is the first bullet point a throughout the section kind of update? If so, I would format it as above for clarity.

# July 26, 2024

## Flutter SDK

- Sync Device Data: Add information about converting a non-synced realm to a synced realm.

## .NET SDK

- Sync Data:
  - Suspend or Resume a Sync Session: Add additional context to the existing code snippet that demonstrates calling the ``Start()`` and ``Stop()`` methods.

>   - Throughout the section: Update pages to reflect that converting to a Synced Realm is only supported in Partition Bases Sync.

Reviewer: ```suggestion
  - Throughout the section: Update pages to reflect that converting to a Synced Realm is only supported in Partition Based Sync.
```
